cmd/lncli/commands.go

```diff
@@ -1915,6 +1974,13 @@ func payInvoice(ctx *cli.Context) error {
 		return err
 	}
 
+	if !ctx.IsSet("force") {
+		err = confirmPayreq(ctx)
```

Comment:
This will end up parsing the payment request twice, since it's already done above. I would much rather move the parsing of the payment request outside of `confirmPayreq`. This way we can avoid passing in the `cli.Context` and can just pass in the payment request string.